### PR TITLE
feat: support node 18, 20 & 21

### PR DESCRIPTION
**Pull request by termontwouter**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -103,5 +103,8 @@
     "rimraf": "latest",
     "ts-jest": "^29.0.0",
     "typescript": "^5.0.0"
+  },
+  "resolutions": {
+    "@comunica/bus-http": "^2.10.0"
```
Comment (rubensworks) [inline package.json:108]: Is this one really needed? I suspect resetting the lockfile should be sufficient to bump the comunica stack.

**Review by rubensworks — CHANGES_REQUESTED** [review]
Thanks! Just one question.

File: package.json (hunk 1 above)
Comment (termontwouter) [reply package.json]: You are totally right 🤦

**Comment by rubensworks** [comment]
Thanks!

**Pull request merged**